apps/emqx_web_hook/src/emqx_web_hook.erl

```diff
@@ -117,7 +117,7 @@ on_client_connack(ConnInfo = #{clientid := ClientId, username := Username, peern
 %% Client connected
 %%--------------------------------------------------------------------
 
-on_client_connected(#{clientid := ClientId, username := Username, peerhost := Peerhost}, ConnInfo, _Env) ->
+on_client_connected(#{clientid := ClientId, username := Username, peerhost := Peerhost, peerport:= Peerport}, ConnInfo, _Env) ->
```

Comment:
@zmstone 
This line needs to be used with care to match `peerport`, which requires all other protocol plugins to pass the `peerport` field in ClientInfo, including some of the Enterprise Edition plugins.

btw,  we also need to update the `clientinfo` declaration  in  the`emqx_types.hrl` file

Reply:
agree.
we should use `maps:get(peerport, ClientInfo, 0)` to be on the safe side.

the type spec update in emqx_types.erl should give us some level of type check
better than sorry, before we have dialyzer checks everywhere.